microdaq/help/en_US/code_integration/mdaq_block_delete.xml

```diff
@@ -13,9 +13,9 @@ xml:id="mdaq_block_delete">
   </refnamediv>
   <refsynopsisdiv>
 	<title>Description</title>
-	<para>This function deletes all files (C source, macros, images) related with given block name.</para>
+	<para>This function deletes all files (C source, macros, images) related to given block name.</para>
 	<para>
-	
+	If block was removed by accident, copy of source code can be found in etc/tmp/ folder. 
```

Comment:
Change path to: **src\c\userlib\.removed_code\**
Add command that returns path to backup folder: mdaq_toolbox_path() + pathconvert("src\c\userlib\.removed_code")